generate.js

```diff
@@ -19,6 +19,8 @@ fs.readdir(
       const profile = require(path.join(__dirname, "profiles", file.name));
 
       profiles.push(profile);
+
+      profiles.image.alt = "Github profile from "`${profiles.name}`;
```

Comment:
You might want to capitalize the `h` here. What do you think?
```suggestion
      profiles.image.alt = "GitHub profile from "`${profiles.name}`;
```